### PR TITLE
Amls 5874

### PR DESCRIPTION
**Pull request by ajchilts**
Live TCSP issue - Services of another TCSP with no MLR

## Related / Dependant PRs?

N/A

## How Has This Been Tested?

Unit + Local

## Checklist

- [ ] Breaking change (fix or feature that would cause existing functionality to change).
- [x] Build passing.
- [x] Unit tests have full coverage.
- [ ] Unit test approach and implementation has been reviewed with QA (testers).
- [ ] Requires acceptance test run.
- [ ] Appropriate labels added.
- [x] RELEASE NOTES ADDED.

### CODE REVIEW

File: app/models/fe/tcsp/ServicesOfAnotherTCSP.scala [hunk 1]
```diff
@@ -43,16 +41,36 @@ object ServicesOfAnotherTCSP {
 
   }
 
+  def mlrExists(mlrRefNo: Option[String]) = {
+    !mlrRefNo.getOrElse("").isEmpty
```
[the remark this answers is not in the document]
Comment (slavvco) [reply app/models/fe/tcsp/ServicesOfAnotherTCSP.scala:45]: or using below: mlrNo(mlrRefNo).nonEmpty

File: app/models/fe/tcsp/ServicesOfAnotherTCSP.scala [hunk 2]
```diff
@@ -43,16 +41,36 @@ object ServicesOfAnotherTCSP {
...
+  }
+
+  def mlrNo(mlrRefNo: Option[String]) = {
+    mlrRefNo.getOrElse("")
+  }
+
+  def tcspServicesOffered(desView: models.des.SubscriptionView) = {
```
Comment (slavvco) [inline app/models/fe/tcsp/ServicesOfAnotherTCSP.scala:52]: would be nice if method name was saying what it's doing - same for the one below

**Comment by slavvco** [comment]
I've tested this locally and run the regression scenarios with tcsp. Looks good.

**Comment by ajchilts** [comment]
Closing as rebase caused some issues.

**Pull request closed without merging**